README.md

```diff
@@ -8,25 +8,29 @@ The list of providers is published in the form of a statically hosted OPTiMaDe I
 
 If you specifically seek the current list of providers for the latest version of the OPTiMaDe specification, you can access it at this URL:
 
-- [https://www.optimade.org/providers/links/](https://www.optimade.org/providers/links/)
+- [https://www.optimade.org/providers/providers.json](https://www.optimade.org/providers/providers.json)
```

Comment:
The previous URI, 'https://www.optimade.org/providers/links', was format-neutral (and permitted both content negotiation and arbitrary additional formats). JSON is mostly useful for machine-readable content, but an obvious additional would be (X)HTML for convenient viewing in browsers, suggesting 'https://www.optimade.org/providers/links.html' along with 'https://www.optimade.org/providers/links.json'.

With the `https://www.optimade.org/providers/providers.json` URI, the format is fixed for JSON. Does it also support (auto-generated)  `https://www.optimade.org/providers/providers.html` in the future? And what about format-agnostic, content-negotiated URI? It would be `https://www.optimade.org/providers/providers`, which has an awkward repetition.

Reply:
See my general comment about the providers.json name. Given that, do you still think we should name it /providers? I do see a point in making this URI as stable as possible. Or should we simply remove it? The setup here is intend to keep `/v1/links` updated with the full list of providers even when we are at far higher versions, only it will stay on the v1 format. It just means that people who use that link to direct other people to see our present list of providers may not see all meta-data we have on those providers (if that is part of a newer, non-backward compatible version of the format).